biztown/src/components/AboutSection.js

```diff
@@ -1,241 +1,298 @@
 import { Component } from "react";
```

Comment:
small stylistic thing: on Firefox, before selecting submit question 3 and 6 already appear to have the red.
<img width="771" alt="Screen Shot 2021-03-17 at 1 44 23 AM" src="https://user-images.githubusercontent.com/28936879/111420863-5a32fb00-86c2-11eb-9e95-0fda56cc1573.png">

Reply:
It appears the behaviour is only restricted to this page which is good. :)

Reply:
Added to backlog.